src/charts.js

```diff
@@ -348,6 +368,45 @@ function showTotalDownloads(sanitizedData, fromDate, toDate, showSum) {
             + dateToDayKey(toDate) + '</em>:');
 }
 
+function getDownloadData(packageNames, fromDate, toDate) {
+    return new Promise(function (accept, reject) {
```

Comment:
ES5 also doesn't have promises.

Reply:
Fixed by bringing in: https://www.npmjs.com/package/promise

Reply:
Okay, actually it's far better to use a ponyfill (which will use inbuilt Promise if available and a polyfill otherwise) instead of using an API implementation.

So, I changed this to bring in: https://github.com/floatdrop/pinkie-promise